app/main.py

```diff
@@ -1,2 +1,54 @@
+from typing import Any, Hashable
+from dataclasses import dataclass
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    key_hash: int
```

Comment:
The `key_hash` attribute in the `Node` class stores the hash of the key at the time of node creation. However, during resizing, the hash function might change (though unlikely with Python's built-in hash), or the hash value might need to be recalculated for other reasons. Consider recalculating the hash during resizing to ensure consistency.